.github/workflows/go_tests.yml

```diff
@@ -49,5 +49,7 @@ jobs:
           flags: go 
           files: ./sdks/go/pkg/coverage.txt
           name: go-unittests
+      - name: Run fmt
+        run: cd sdks/go/pkg/beam && go fmt ./...; git diff-index --quiet HEAD || (echo "Run go fmt before checking in changes" && exit 1)
```

Comment:
Rather than using `cd`, you can specify a working-directory (see https://github.com/apache/beam/blob/b07840e925b910fffee85d3bcdb67468965b7770/.github/workflows/python_tests.yml#L77-L79 for an example). Its not a bad idea to update the rest of the file, but I'll leave that to your discretion.